p2panda-rs/src/document/document.rs

```diff
@@ -358,7 +361,8 @@ mod tests {
     }
 
     #[rstest]
-    fn resolve_documents(schema: SchemaId) {
+    #[async_std::test]
+    async fn resolve_documents(schema: SchemaId) {
```

Comment:
There are other places where we use `schema` to mean `schema_id` (like in `Document`), maybe this could be an issue to correct/decide on what we want, in all of these places?

Reply:
Oh ja, that would be nice!